metabase/metabase
# Add card field values preview

Epic https://github.com/metabase/metabase/issues/26898

How to test:
- Add a question based on the `Products` table
- Open a dashboard with a card
- Edit -> Filter -> Text -> Dropdown -> Edit
- Connect the parameter to the card
- Click `Dropdown list` -> `Edit` in the parameter sidebar
- Click `From another question or model` and pick the question based on products with `Category` as the column
- Make sure you can see preview of selected values 

<img width="1153" alt="Screenshot 2023-01-11 at 16 17 52" src="https://user-images.githubusercontent.com/8542534/211829592-d10174ee-4d1a-40b4-8f0f-7880c2b819ac.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27627)
